deno.jsonc

```diff
@@ -26,8 +26,8 @@
     }
   },
   "tasks": {
-    "test": "deno fmt --check && deno lint && deno bundle src/mod.ts && deno test --allow-read",
-    "coverage": "deno test --allow-read --coverage=.coverage && deno coverage --exclude=fixtures --exclude=test --lcov --output=lcov.info .coverage && deno run --allow-read https://deno.land/x/code_coverage@0.2.0/cli.ts"
+    "test": "deno fmt --check && deno lint && deno bundle src/mod.ts && deno test --allow-read --parallel",
+    "coverage": "deno test --reporter=dot --parallel --allow-read --coverage=.coverage && deno coverage --exclude=fixtures --exclude=test --lcov --output=lcov.info .coverage && deno run --allow-read https://deno.land/x/code_coverage@0.2.0/cli.ts"
```

Comment:
Do we need the reporter change here? The `deno run code_coverage` invocation at the end handles reporting.

Reply:
Yes exactly the default report will print the test results but `--reporter=dot` will hide them as follows

`deno task coverage`

```txt
Task coverage rm -rf .coverage && deno test --reporter=dot --parallel --allow-read --coverage=.coverage src/  && deno coverage --exclude=fixtures --exclude=test --lcov --output=lcov.info .coverage && deno run --allow-read https://deno.land/x/code_coverage@0.2.0/cli.ts
..................................................................................................................................
..................................................................................................................................
..............................
ok | 176 passed (114 steps) | 0 failed (1s)

.------------------------------------------------------------------------------------------------------------------------------------------------.
|                              File Path                              | Coverage |                    Lines Without Coverage                     |
|---------------------------------------------------------------------|----------|---------------------------------------------------------------|
```

Reply:
Ohhhh that's awesome. I like it!